examples/generative/corrdiff/generate.py

```diff
@@ -81,36 +83,32 @@ def main(cfg: DictConfig) -> None:
     S_noise = getattr(cfg, "S_noise", 1)
 
     # Parse data options
-    train_data_path = getattr(cfg, "train_data_path")
+    times_range = getattr(cfg, "time_range", None)
     patch_size = getattr(cfg, "patch_size", 448)
-    crop_size_x = getattr(cfg, "crop_size_x", 448)
-    crop_size_y = getattr(cfg, "crop_size_y", 448)
-    n_history = getattr(cfg, "n_history", 0)
-    in_channels = getattr(
-        cfg, "in_channels", [0, 1, 2, 3, 4, 9, 10, 11, 12, 17, 18, 19]
-    )
-    out_channels = getattr(cfg, "out_channels", [0, 17, 18, 19])
-    img_shape_x = getattr(cfg, "img_shape_x", 448)
-    img_shape_y = getattr(cfg, "img_shape_y", 448)
     patch_shape_x = getattr(cfg, "patch_shape_x", 448)
     patch_shape_y = getattr(cfg, "patch_shape_y", 448)
     overlap_pix = getattr(cfg, "overlap_pix", 0)
     boundary_pix = getattr(cfg, "boundary_pix", 2)
-    roll = getattr(cfg, "roll", False)
-    add_grid = getattr(cfg, "add_grid", True)
-    ds_factor = getattr(cfg, "ds_factor", 1)
-    min_path = getattr(cfg, "min_path", None)
-    max_path = getattr(cfg, "max_path", None)
-    global_means_path = getattr(cfg, "global_means_path", None)
-    global_stds_path = getattr(cfg, "global_stds_path", None)
-    gridtype = getattr(cfg, "gridtype", "sinusoidal")
-    N_grid_channels = getattr(cfg, "N_grid_channels", 4)
-    normalization = getattr(cfg, "normalization", "v2")
+
+    if times_range is not None:
+        times = []
+        t_start = datetime.datetime.strptime(times_range[0], time_format)
+        t_end = datetime.datetime.strptime(times_range[1], time_format)
+        dt = datetime.timedelta(hours=(times_range[2] if len(times_range) > 2 else 1))
+        times = [
+            t.strftime(time_format)
+            for t in time_range(t_start, t_end, dt, inclusive=True)
+        ]
```

Comment:
I think the line bellow sets times from the cfg instead of using this unless I am mistaken. Meaning `times` is just overwritten with the following line, `    times = getattr(cfg, "times", ["2021-02-02T00:00:00"])
￼`

Reply:
Yeah, sorry, that was dumb. Need to put that inside an `else` block (which it was originally).